test/controllers/NCTSMessageControllerSpec.scala

```diff
@@ -281,6 +287,7 @@ class NCTSMessageControllerSpec extends SpecBase with ScalaCheckPropertyChecks w
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "Internal Submission Failure Future(Success(SubmissionFailureInternal))"
```

Comment:
The code should be amended to return a status code only. Any internal failures should be logged.